packages/core/src/__tests__/connectSaga.spec.js

```diff
@@ -12,29 +12,24 @@ beforeEach(() => {
   const sagaMiddleware = createSagaMiddleware();
 
   store = createStore(
-    ({ actionTypes = [] } = {}, { type }) => {
-      if (/^DIRECT_LINE\/(DIS|RE)?CONNECT/.test(type)) {
+    ({ actionTypes = [] } = {}, { type }) =>
+      /^DIRECT_LINE\/(DIS|RE)?CONNECT/u.test(type) ?
         // We are only interested in CONNECT_*, RECONNECT_*, and DISCONNECT_* actions.
-
-        return {
-          actionTypes: [...actionTypes, type]
-        };
-      } else {
-        return { actionTypes };
-      }
-    },
+        { actionTypes: [...actionTypes, type] }
+      :
+        { actionTypes },
     applyMiddleware(sagaMiddleware)
   );
 
   sagaMiddleware.run(connectSaga);
 
   directLine = {
-    activity$: new Observable(() => {}),
+    activity$: new Observable(() => () => 0),
```

Comment:
why?

Reply:
The function passed to `Observable` need to return another function that is called, when the `activity$` is unsubscribed.

```js
new Observable(() => {
  console.log('Someone just subscribed me.');

  return () => console.log('That guy just unsubscribed me.');
});
```